City-of-Helsinki/drupal-hdbt
# UHF-5915: Move tag styles to mixins for ease-of-reuse

# [UHF-5915](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5915)
Moved tag styles to mixins to make them easier to use with React components.

## What was done
* Moved tag styles to mixins
* Added style for search applications tags
* Made existing tags use the mixins

## How to install
* `git checkout origin/UHF-5915-filter-pills`
* Read install instructions here: https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/116

## How to test
* Check that the new search app tag style looks correct
* Check that code makes sense

## Designers review



## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/116
